src/test/java/api/support/fixtures/LoansFixture.java

```diff
@@ -130,6 +134,12 @@ public IndividualResource checkOutByBarcode(IndividualResource item)
     return checkOutByBarcode(item, usersFixture.jessica());
   }
 
+  public Response batchChangeDueDate(BatchChangeDueDateRequestBuilder requestBuilder) {
+
+    return restAssuredClient.post(requestBuilder.create(),
+      InterfaceUrls.batchChangeDueDateUrl(), "batch-change-due-date-request");
```

Comment:
Personally, I would use a static import of the `batchchangeDueDateUrl` method to improve the readability of this line

```suggestion
     batchChangeDueDateUrl(), "batch-change-due-date-request");
```